### PR TITLE
Remove default `jupyter.<domain>` from url and use domain

### PR DESCRIPTION
**Pull request by costrouc**

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -145,7 +145,8 @@ jobs:
         run: |
           mkdir -p local-deployment
           cd local-deployment
-          qhub init local --project=thisisatest  --domain github-actions.qhub.dev --auth-provider=password --terraform-state=local
+          qhub init local --project=thisisatest  --domain jupyter.github-actions.qhub.dev --auth-provider=password --terraform-state=local
```
Comment (aktech) [inline .github/workflows/test.yaml:148]: Do we need `jupyter.` in here?

Comment (costrouc) [reply .github/workflows/test.yaml:148]: No we don't I only did this for consistency. Should we just use `github-actions.qub.dev` so as to set the example that `jupyter` is no longer required?

Comment (aktech) [reply .github/workflows/test.yaml:148]: I think that will be great.

**Comment by aktech** [comment]
@costrouc Thanks a lot for this!

**Pull request merged**